src/components/Code.astro

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
Created this component to avoid having `div` tags in the markdown files. This component will be filled out once we add styling as per the [Figma design](https://www.figma.com/file/IgRlQWJTyGoZhlF4fVKD6B/Centrapay-Docs-(Tailwind)?type=design&node-id=1654-8412&mode=design&t=AEMFKFrkAbgapcAS-0)

Reply:
@AndyClifford what styling would end up here?
We could perhaps rename the component to be `CodeColumn` and have the `<Column sticky>` tags here.

Reply:
Yeah sounds good. The Protocol example uses `CodeGroup`?

Reply:
@AndyClifford yeah perhaps let's rename to `CodeGroup` 👍

Reply:
Do you think we will always have code examples i.e. need two columns? If not maybe having the code group responsible for creating the second column is the right play.

Reply:
@AndyClifford I think we always would have code examples for an endpoint.